### PR TITLE
refactor the volume-snapshot restore performance description

### PR DESCRIPTION
**Pull request by fengou1**
### What is changed, added, or deleted? (Required)
https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/backup-restore-snapshot-perf description may confuse the user.

1. why tikv startup takes so longs?
[solution]

- take v6.5.0 test result. 5 times test result, data size from 50GB, 100GB, 200GB, 500GB, 1.4TB. max tikv start time 16 minutes. min 10 minutes.
- explain more regarding the tikv startup so long.

3. why the restore perf curve is not linear.
explain the reason for the affection.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/backup-restore-snapshot-perf.md [hunk 1]
```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
...
 | 恢复阶段     | 恢复大致耗时 | 恢复总占比 | 备注                                                                            |
 | :--------: | :---------: | :------: | :----------------------------------------------------------------------------: |
-| 创建集群     | 30s         | 1%      | 包含下载 docker image 和启动 PD 的时间                                                 |
-| 卷恢复      | 20s         | 1%     | 包含启动 BR pod 和卷恢复的时间                                                         |
-| 启动 TiKV   | 2～30 min    | 58%      | TiKV 启动时长受卷性能影响，一般 3 分钟左右，当恢复数据包含事务时，此阶段耗时可能达到 30 分钟 |
-| 数据恢复阶段 | 2～20 min    | 38%       | 包含恢复 raft 共识层数据以及删除 MVCC 数据的时间                                      |
-| 启动 TiDB   | 1 min        | 2%       | 包含下载 docker image 和启动 TiDB 的时间                                           |
+| 创建集群     | 30 s         | 2%      | 包含下载 docker image 和启动 PD 的时间                                             |
+| 卷恢复      | 20 s         | 1%       | 包含启动 BR pod 和卷恢复的时间                                                     |
+| 启动 TiKV   | ~16 min    | 42%       | 包含 TiKV 启动时，启动 rocksdb，读取所有 region 元信息。                             |
```
Comment (3pointer) [inline zh/backup-restore-snapshot-perf.md:89]: `~` seems different with `～`

File: zh/backup-restore-snapshot-perf.md [hunk 2]
```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
...
 
 > **注意：**
 >
-> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复阶段 TiKV 启动阶段，需要数据自检和自动修复。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复，如提升 IOPS 和带宽, 额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
+> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复阶段 TiKV 启动阶段和数据恢复阶段，需要初始化数据，即从 S3 下载数据。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复，如提升 IOPS 和带宽, 额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
 
 ### 恢复性能数据
 
-卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动受 EBS 卷快照恢复延迟初始化影响，当前 AWS 并没有提供恢复卷延迟初始完成量化指标。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
+卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动和数据恢复都需要读取恢复卷数据。对于从快照恢复的卷，数据必须先从 Amazon S3 下载数据块并将其写入到恢复的卷中，然后才能访问这些数据。该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据我们的测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```
Comment (3pointer) [inline zh/backup-restore-snapshot-perf.md:99]:
```suggestion
卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动和数据恢复都需要读取卷数据。通过快照恢复的卷具有延迟加载的特性，即刚刚恢复完成后不能立刻达到最佳性能，原因是数据必须等数据从 Amazon S3 下载完成后并将其写入到恢复的卷中，后续才能正常访问这些数据。该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据我们的测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```

**Review by 3pointer — COMMENTED** [review]
rest LGTM

File: zh/backup-restore-snapshot-perf.md (hunk 1 above)
Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: check the recently 5 times restore, time consume from 10 minutes to 16 minutes for tikv startup

File: zh/backup-restore-snapshot-perf.md [hunk 3]
```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
 
 | 恢复阶段     | 恢复大致耗时 | 恢复总占比 | 备注                                                                            |
 | :--------: | :---------: | :------: | :----------------------------------------------------------------------------: |
-| 创建集群     | 30s         | 1%      | 包含下载 docker image 和启动 PD 的时间                                                 |
-| 卷恢复      | 20s         | 1%     | 包含启动 BR pod 和卷恢复的时间                                                         |
-| 启动 TiKV   | 2～30 min    | 58%      | TiKV 启动时长受卷性能影响，一般 3 分钟左右，当恢复数据包含事务时，此阶段耗时可能达到 30 分钟 |
-| 数据恢复阶段 | 2～20 min    | 38%       | 包含恢复 raft 共识层数据以及删除 MVCC 数据的时间                                      |
-| 启动 TiDB   | 1 min        | 2%       | 包含下载 docker image 和启动 TiDB 的时间                                           |
+| 创建集群     | 30 s         | 2%      | 包含下载 docker image 和启动 PD 的时间                                             |
+| 卷恢复      | 20 s         | 1%      | 包含启动 BR pod 和卷恢复的时间                                                     |
```
Comment (shichun-0415) [inline zh/backup-restore-snapshot-perf.md:88]:
```suggestion
| 创建集群     | 30s         | 2%      | 包含下载 docker image 和启动 PD 的时间                                             |
| 卷恢复      | 20s         | 1%      | 包含启动 BR pod 和卷恢复的时间                                                     |
```
秒 (s) 跟数字之间不加空格

File: zh/backup-restore-snapshot-perf.md [hunk 4]
```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
...
+| 启动 TiKV   | 10~16 min    | 42%     | 包含 TiKV 启动时，启动 rocksdb，读取所有 region 元信息。                             |
```
Comment (shichun-0415) [inline zh/backup-restore-snapshot-perf.md:89]:
```suggestion
| 启动 TiKV   | 10～16 min    | 42%     | 包含 TiKV 启动时，启动 RocksDB 及读取所有 Region 元信息的时间。                             |
```

File: zh/backup-restore-snapshot-perf.md [hunk 5]
```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
...
+| 数据恢复阶段 | 2～20 min    | 52%      | 包含恢复 raft 共识层数据以及删除 MVCC 数据的时间                                     |
+| 启动 TiDB   | 1 min        | 3%      | 包含下载 docker image 和启动 TiDB 的时间                                          |
 
 > **注意：**
 >
-> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复阶段 TiKV 启动阶段，需要数据自检和自动修复。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复，如提升 IOPS 和带宽, 额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
+> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复阶段 TiKV 启动阶段和数据恢复阶段，需要初始化数据，即从 S3 下载数据。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复，如提升 IOPS 和带宽, 额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
```
Comment (shichun-0415) [inline zh/backup-restore-snapshot-perf.md:95]:
```suggestion
> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复时，启动 TiKV 和恢复数据需要初始化数据，即从 Amazon S3 下载数据。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复或提升 IOPS 和带宽，额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
```

File: zh/backup-restore-snapshot-perf.md [hunk 6]
```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
...
 
 ### 恢复性能数据
 
-卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动受 EBS 卷快照恢复延迟初始化影响，当前 AWS 并没有提供恢复卷延迟初始完成量化指标。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
+卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动和数据恢复都需要读取卷数据。通过快照恢复的卷具有延迟加载的特性，即刚刚恢复完成后不能立刻达到最佳性能，原因是数据必须等数据从 Amazon S3 下载完成后并将其写入到恢复的卷中，后续才能访问这些数据。该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据我们的测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```
Comment (shichun-0415) [inline zh/backup-restore-snapshot-perf.md:99]:
```suggestion
卷快照恢复时间主要取决于启动 TiKV 和数据恢复消耗的时间，启动 TiKV 和恢复数据都需要读取卷数据。通过快照恢复的卷具有延迟加载的特性，即刚刚恢复完成后，不能立刻达到最佳性能。这是因为，从 Amazon S3 下载完数据后，还需要等数据写入到恢复的卷中，这样，后续才能访问这些数据。该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```

File: zh/backup-restore-snapshot-perf.md [hunk 7]
```diff
@@ -107,3 +107,7 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
 | 200 GB   | 500 GB  | 7000IOPS/400 MiB/s | 21 min   |
 | 500 GB   | 1024 GB | 7000IOPS/400 MiB/s | 25 min   |
 | 1024 GB  | 3500 GB | 7000IOPS/400 MiB/s | 34 min   |
+
+> **注意：**
+>
+> 恢复数据性能曲线并不是完全线性单调递增，这里卷数据指的是 TiKV 实际使用的用户数据，而 TiKV 启动和数据恢复阶段实际加载和扫描的卷数据块可能不同，同时受 AWS 卷数据块下载网络速度等的影响，导致恢复数据性能曲线并不完全线性单调递增。
```
Comment (shichun-0415) [inline zh/backup-restore-snapshot-perf.md:113]:
```suggestion
> 数据恢复性能曲线并非完全线性单调递增，这是因为，卷数据是 TiKV 实际使用的用户数据，而启动 TiKV 和恢复数据时，可能需要加载和扫描不同的卷数据块，另外，AWS 卷数据块下载网络速度等因素对恢复性能也会产生影响。
```

Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44

File: zh/backup-restore-snapshot-perf.md (hunk 6 above)
Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44

File: zh/backup-restore-snapshot-perf.md (hunk 5 above)
Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44

File: zh/backup-restore-snapshot-perf.md (hunk 4 above)
Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44

File: zh/backup-restore-snapshot-perf.md (hunk 3 above)
Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44

File: zh/backup-restore-snapshot-perf.md [hunk 8]
```diff
@@ -82,28 +82,32 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
...
 
 > **注意：**
 >
-> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复阶段 TiKV 启动阶段，需要数据自检和自动修复。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复，如提升 IOPS 和带宽, 额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
+> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复时，启动 TiKV 和恢复数据需要初始化数据，即从 Amazon S3 下载数据。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复或提升 IOPS 和带宽，额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
 
 ### 恢复性能数据
 
-卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动受 EBS 卷快照恢复延迟初始化影响，当前 AWS 并没有提供恢复卷延迟初始完成量化指标。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
+卷快照恢复时间主要取决于启动 TiKV 和数据恢复阶段，启动 TiKV 和恢复数据都需要读取卷数据。通过快照恢复的卷具有延迟加载的特性，即刚刚恢复完成后，不能立刻达到最佳性能。这是因为，数据必须等数据从 Amazon S3 下载完成后并将其写入到恢复的卷中，后续才能访问这些数据。该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```
Comment (shichun-0415) [inline zh/backup-restore-snapshot-perf.md:99]:
```suggestion
卷快照恢复时间主要取决于启动 TiKV 和数据恢复阶段，启动 TiKV 和恢复数据都需要读取卷数据。通过快照恢复的卷会延迟加载，即刚刚恢复完成后，不能立刻达到最佳性能。这是因为，数据只有从 Amazon S3 下载完成并写入到恢复的卷之后，后续才能正常访问。

该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```

File: zh/grant-permissions-to-remote-storage.md [hunk 9]
```diff
@@ -130,6 +130,8 @@ kubectl create secret generic s3-secret --from-literal=access_key=xxx --from-lit
      kubectl annotate sa tidb-controller-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=tidb-admin
      ```
 
+     重启 tidb-controller-manager Pod, 使得 ServiceAccount tidb-controller-manager 生效。
```
Comment (shichun-0415) [inline zh/grant-permissions-to-remote-storage.md:133]:
```suggestion
     重启 tidb-controller-manager Pod，使得 ServiceAccount tidb-controller-manager 生效。
```

Comment (fengou1) [reply zh/grant-permissions-to-remote-storage.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/469d763ab1e94d5c8adea26c36464207d8df77ee

File: zh/backup-restore-snapshot-perf.md (hunk 8 above)
Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/469d763ab1e94d5c8adea26c36464207d8df77ee

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: zh/grant-permissions-to-remote-storage.md [hunk 10]
```diff
@@ -130,6 +130,8 @@ kubectl create secret generic s3-secret --from-literal=access_key=xxx --from-lit
      kubectl annotate sa tidb-controller-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=tidb-admin
      ```
 
+     重启 tidb-controller-manager Pod，使得 ServiceAccount tidb-controller-manager 生效。
```
Comment (3pointer) [inline zh/grant-permissions-to-remote-storage.md:133]: 这句话感觉不太通顺

Comment (fengou1) [reply zh/grant-permissions-to-remote-storage.md]: fixed https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/92fa9cd049e320ebfdde5ca5002a728d6453e8db

File: zh/grant-permissions-to-remote-storage.md [hunk 11]
```diff
@@ -130,6 +130,8 @@ kubectl create secret generic s3-secret --from-literal=access_key=xxx --from-lit
      kubectl annotate sa tidb-controller-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=tidb-admin
      ```
 
+     重启 TiDB Operator 的 tidb-controller-manager Pod。使得配置的 ServiceAccount 生效。
```
Comment (shichun-0415) [inline zh/grant-permissions-to-remote-storage.md:133]:
```suggestion
     重启 TiDB Operator 的 tidb-controller-manager Pod，使配置的 ServiceAccount 生效。
```

File: zh/backup-restore-snapshot-perf.md (hunk 1 above)
Comment (fengou1) [reply zh/backup-restore-snapshot-perf.md]: > `~` seems different with `～`

fixed

**Review by 3pointer — APPROVED** [review]

**Pull request merged**